charts/sourcegraph/values.yaml

```
@@ -729,7 +731,9 @@ prometheus:
     allowPrivilegeEscalation: false
     runAsUser: 100
     runAsGroup: 100
-    readOnlyRootFilesystem: true
+    # -- Read-only filesystem not supported for the prometheus container,
```

```suggestion
    # Read-only filesystem not supported for the prometheus container,
```
I don't think these need to generate an individual entry in the README